### PR TITLE
Support button

### PR DESCRIPTION
**Pull request by rukawa444**
Added a contact email on support navigation and cypress test.
support mail

### CODE REVIEW

**Review by mentor-tara — APPROVED** [review]
Go go go :smile:[](prolog-9)

**Comment by rukawa444** [comment]
In the cypress test. it can actually open the mailto window. The concern is the timeout for loading the page, so it keeps getting red...

File: features/ui/sidebar-navigation/sidebar-navigation.tsx [hunk 1]
```diff
@@ -191,11 +191,12 @@ export function SidebarNavigation() {
           </LinkList>
 
           <List>
-            <MenuItemButton
+            <MenuItemLink
               text="Support"
               iconSrc="/icons/support.svg"
               isCollapsed={isSidebarCollapsed}
-              onClick={() => alert("Support")}
+              href="mailto:support@prolog-app.com?subject='Support Request:'"
```
Comment (jkettmann) [inline features/ui/sidebar-navigation/sidebar-navigation.tsx:198]: Exactly 😄

File: cypress/integration/support-mail.spec.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    cy.visit("http://localhost:3000/dashboard");
+  });
+
+  context("desktop resolution", () => {
+    beforeEach(() => {
+      cy.viewport(1920, 1080);
+    });
+
+    it("opens the mail app", () => {
+      cy.get("nav").contains("Support").click({ force: true });
```
Comment (jkettmann) [inline cypress/integration/support-mail.spec.ts:12]: I don't think you can test that the email app actually opens. What you can do though is check that the link has the correct `href`

File: features/ui/sidebar-navigation/sidebar-navigation.tsx (hunk 1 above)
Comment (rukawa444) [reply features/ui/sidebar-navigation/sidebar-navigation.tsx:198]: sorry for that! I got lost in the wild lol!✌

**Comment by rukawa444** [comment]
closing in favor #10.

**Pull request closed without merging**